### PR TITLE
PLAT-238: Don't change pulse in insolar net

### PR DESCRIPTION
**Pull request by ivanshibitov**
…luanch pulsar with option --one-shot. 3. Increase PulseWatchdogTimeout up to 1 month

**- What I did**
1. Func tests run luanchnet.sh without pulsar. 
2. Func tests run pulsar with option --one-shot. 
3. Increase PulseWatchdogTimeout up to 1 month to prevent crash of nodes

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: ledger-core/v2/application/testutils/launchnet/launchnet.go [hunk 1]
```diff
@@ -404,6 +404,34 @@ func waitForNet() error {
...
+	output, err := pulsarCmd.CombinedOutput()
+	fmt.Println("Pulsar launch output: ", string(output))
+
+	return errors.Wrap(err, "failed to launch pulsar")
+}
+
+func waitForNet() error {
+	err := waitForNetworkState(insolar.WaitPulsar)
+	if err != nil {
+		return errors.Wrap(err, "Can't for NetworkState "+insolar.WaitPulsar.String())
```
Comment (ElSacramento) [inline ledger-core/v2/application/testutils/launchnet/launchnet.go:423]: Can't get NetworkState? Can't wait for state? what can't do?

File: ledger-core/v2/application/testutils/launchnet/launchnet.go [hunk 2]
```diff
@@ -404,6 +404,34 @@ func waitForNet() error {
...
+	}
+
+	err = runPulsar()
+	if err != nil {
+		return errors.Wrap(err, "Can't run pulsar")
+	}
+
+	err = waitForNetworkState(insolar.CompleteNetworkState)
+	if err != nil {
+		return errors.Wrap(err, "Can't for NetworkState "+insolar.WaitPulsar.String())
```
Comment (ElSacramento) [inline ledger-core/v2/application/testutils/launchnet/launchnet.go:433]: `insolar.CompleteNetworkState.String()`

File: ledger-core/v2/cmd/pulsard/main.go [hunk 3]
```diff
@@ -104,6 +106,18 @@ func main() {
...
+
+	if params.oneShot {
+		nextPulseNumber := pulse.OfNow()
+		err := server.Send(ctx, nextPulseNumber)
+		if err != nil {
+			panic(err)
+		}
+		// it's required since pulse is sent in goroutine
+		time.Sleep(time.Second * 10)
+		return
```
Comment (ElSacramento) [inline ledger-core/v2/cmd/pulsard/main.go:118]: I see `err = cm.Stop(ctx)` in defer. Do we need to stop component manager in one-shot run?

Comment (ivanshibitov) [reply ledger-core/v2/cmd/pulsard/main.go:118]: There is no any action we can do if it's not stopped.
And it's the end of the program.
And code is shorter.
That is why I didn't call Stop().
But, for perfectness - yes.

**Pull request merged**